app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+
+
+def copy_file(command: str) -> None:
+    done_lst_path = command.split()
+
+    if done_lst_path[1] == done_lst_path[2]:
+        return
+
+    if done_lst_path[0] != "cp":
+        print("Wrong command use 'cp'.")
+        return
+
+    if not os.path.exists(done_lst_path[1]):
+        print(f"File {done_lst_path[1]} not exist.")
+        return
```

Comment:
Code Efficiency Issue: You should check if the source file exists before checking if the source and destination paths are the same. This way, you can return early and avoid unnecessary operations if the file does not exist.